lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -89,11 +92,13 @@ sub group_overview {
         }
     }
     $self->stash('result',          $res);
-    $self->stash('group',           $group);
     $self->stash('limit_builds',    $limit_builds);
     $self->stash('only_tagged',     $only_tagged);
     $self->stash('comments',        \@comments);
     $self->stash('pinned_comments', \@pinned_comments);
+    $self->respond_to(
+        json => {json     => {result => $res}},
```

Comment:
I already had the same idea but then I thought this is beyond of the problem this feature tries to solve. We have comments API anyways.

But if you like I include it. I would have to resolve the DB queries in the controller and adjust the view code.